plugins/module_utils/foreman_helper.py

```diff
@@ -51,6 +51,7 @@
     'remote_execution': 'remote_execution_features',
     'scc_manager': 'scc_accounts',
     'snapshot_management': 'snapshots',
+    'foreman_templates': 'templates',
```

Comment:
FWIW, we usually skip the `foreman_` prefix for the other plugins.

Reply:
here you go